gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsFragment.java

```diff
@@ -421,9 +421,14 @@ public void enableCurrentLocationIndicator() {
 
   @Override
   public void setMapFeatures(ImmutableSet<MapFeature> features) {
-    Timber.d("Set map features called : %s", features.size());
+    if (features.isEmpty()) {
```

Comment:
Is this correct? I feel this might be wrong because it would prevent cleaning up the map if the new activated project has 0 features.

Reply:
Fixed.